api/capi/src/nnstreamer-capi-util.c

```diff
@@ -777,3 +815,42 @@ ml_check_nnfw_availability (ml_nnfw_type_e nnfw, ml_nnfw_hw_e hw,
 done:
   return ML_ERROR_NONE;
 }
+
+/**
+ * @brief Checks whether machine_learning.inference feature is enabled or not.
+ */
+int ml_get_feature_enabled (void)
+{
+  /** -1: Not checked yet, 0: Not supported, 1: Supported */
+  static int feature_enabled = -1;
+
+/**
+ * In case of Unit Test or Non-Tizen API, then feature checking is not necessary.
+ * So it always returns ML_ERROR_NONE;
+ */
+#if defined(__UNITTEST__)
```

Comment:
The option '-Denable-unittest=true' defines '__UNITTEST__' and this always makes the return result ML_ERROR_NONE.

Reply:
Not necessary.